ion-schema/src/isl/util.rs

```diff
@@ -103,6 +105,20 @@ impl TimestampPrecision {
             },
         }
     }
+
+    fn string_value(&self) -> String {
```

Comment:
🗺️ By moving the variant-to-string mapping here, we are able to call this function from the `Display` impl and the `WriteToIsl` impl.